### ISSUE TITLE
Thoughts on making a fixed-point physics library

### ISSUE THREAD

**Issue by BenMcLean** [nilpunch/fixed-point#1]
I would recommend that a C# fixed point physics library should be made as a .NET Standard 2.0 library for maximum interoperability with all newer .NET versions. It could optionally take advantage of newer C# version features with [PolySharp](https://github.com/Sergio0694/PolySharp) but should ultimately compile to .NET Standard 2.0, should be completely .NET managed (no native wrappers) so it can build anywhere, should be MIT-licensed and should obviously absolutely not depend on the Unity framework at all. I'd be really interested in a project taking that approach.

**Comment by nilpunch** [comment]
Yeah, I’m on the same page. This repo is currently on hold, but I’ll return to it after completing the core prediction-rollback functionality in the other library.

Thanks for sharing your thoughts!
